simplq/.gitignore

```diff
@@ -18,6 +18,9 @@
 .env.test.local
 .env.production.local
 
+
+#json
+.eslintrc.json
 npm-debug.log*
```

Comment:
We can't ignore eslintrc. A change might be required later that should be followed by everyone. So remove it from gitignore, and remove the endOfLine change and make VSCode ignore the error you're getting just within that particular file.

Reply:
![Screenshot from 2020-10-06 13-15-22](https://user-images.githubusercontent.com/12583528/95173366-32b43a00-07d6-11eb-84c2-0ba9f4e0abce.png)

Reply:
In My VScode its showing error at everyline. Thats why i did that . but okay I'll keep that as it was

Reply:
Check now